### PR TITLE
Личный проект: подробности

### PR DESCRIPTION
**Pull request by akzhar**
---
:mortar_board: [Личный проект: подробности](https://up.htmlacademy.ru/javascript/17/user/490939/tasks/7)

:boom: https://htmlacademy-javascript.github.io/490939-keksobooking-17/3/

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -57,7 +57,39 @@ function getFragmentWithPins(apartments) {
...
+function setAddressCoordinates(pinObj) {
+  var addressInput = document.querySelector('#address');
+  var x = pinObj.style.left.slice(0, -2);
+  var y = pinObj.style.top.slice(0, -2);
+  addressInput.value = x + ', ' + y;
+}
+
+function mapPinMainClickHandler() {
+  var mapBlock = document.querySelector('.map');
+  var adForm = document.querySelector('.ad-form');
```
Comment (rkhaslarov) [inline js/main.js]: Should be got once

File: js/main.js [hunk 2]
```diff
@@ -57,7 +57,39 @@ function getFragmentWithPins(apartments) {
...
+  var mapPins = mapBlock.querySelector('.map__pins');
+  var apartments = createApartments(NUMBER_OF_OFFERS);
+  var fragment = getFragmentWithPins(apartments);
+  mapPins.appendChild(fragment);
+  mapBlock.classList.remove('map--faded');
+  adForm.classList.remove('ad-form--disabled');
```
Comment (rkhaslarov) [inline js/main.js]: adForm.classList.remove('ad-form--disabled'); it's part of unlocking form, isn't it?

File: js/main.js [hunk 3]
```diff
@@ -57,7 +57,39 @@ function getFragmentWithPins(apartments) {
...
+  addressInput.value = x + ', ' + y;
+}
+
+function mapPinMainClickHandler() {
+  var mapBlock = document.querySelector('.map');
+  var adForm = document.querySelector('.ad-form');
+  var mapPins = mapBlock.querySelector('.map__pins');
+  var apartments = createApartments(NUMBER_OF_OFFERS);
+  var fragment = getFragmentWithPins(apartments);
+  mapPins.appendChild(fragment);
```
Comment (rkhaslarov) [inline js/main.js]: Each click duplicates the pins

File: js/main.js [hunk 4]
```diff
@@ -57,7 +57,39 @@ function getFragmentWithPins(apartments) {
...
+  removeAttributes(inputs, 'disabled');
+  removeAttributes(selects, 'disabled');
+  sumbitBtn.removeAttribute('disabled');
+}
+
+function setAddressCoordinates(pinObj) {
+  var addressInput = document.querySelector('#address');
+  var x = pinObj.style.left.slice(0, -2);
+  var y = pinObj.style.top.slice(0, -2);
+  addressInput.value = x + ', ' + y;
```
Comment (rkhaslarov) [inline js/main.js]: What about the center of the main pin? It wasn't taken into account here as far as I see )

File: js/main.js (hunk 1 above)
Comment (akzhar) [reply js/main.js]: Sure. Fixed

File: js/main.js (hunk 2 above)
Comment (akzhar) [reply js/main.js]: It is, I should pay more attention on such things.

File: js/main.js (hunk 3 above)
Comment (akzhar) [reply js/main.js]: It's king of feature ^^ 
Just kidding. Fixed

File: js/main.js (hunk 4 above)
Comment (akzhar) [reply js/main.js]: fixed

**Pull request merged**